src/test/java/net/imglib2/loops/LoopPerformanceBenchmark.java

```diff
@@ -124,15 +147,18 @@ public void copy_loopBuilder()
 		LoopBuilder.setImages( Views.interval( in, out ), out ).forEachPixel( ( in, out ) -> out.set( in ) );
 	}
 
+	@Benchmark
+	public void copy_flatIterable() {
```

Comment:
There is a bug here. It should be `i = Views.flatIterable( Views.interval( in, out ) ).cursor()`, or `o = Views.flatIterable( allOut ).cursor();`. Probably the latter, if you want to benchmark against best-case performance, where the cursors are actually cursors of the underlying ArrayImgs.